KomaMRIFiles/src/Phantom/Phantom.jl

```diff
@@ -86,13 +86,14 @@ end
 function import_motion!(
     fields::Array, Ns::Int, model::Val{:ArbitraryMotion}, motion_group::HDF5.Group
 )
-    dur = read(motion_group["period_durations"])
+    t_start = read(motion_group["t_start"])
+    t_end = read(motion_group["t_end"])
     K = read_attribute(motion_group, "K")
     dx = zeros(Ns, K - 1)
     dy = zeros(Ns, K - 1)
     dz = zeros(Ns, K - 1)
     for key in HDF5.keys(motion_group)
```

Comment:
That is because at first, the .phantom format was not intended to store the arrays of trajectories for dimensions that have no motion, so you would have to initialize them as arrays of zeros and then assign them the value of the file if it exists. Now I can see that when exporting the motion to a .phantom file, I store all the three matrices even if their values are 0. So exactly, I don´t need a loop now.

Reply:
Ok let's change that.

Reply:
Which name do you want?

Reply:
A name that represents what it is, in the context of ArbitraryMotion.

Reply:
`fields` is not only for ArbitraryMotion, it contains all the parameters (all the "fields") of the phantom. It's not crucial but, honestly, I don't know which name fits better than that.

Reply:
phantom_fields

Reply:
Solved in [5bc95ef](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/5bc95ef0006efec78b5cbe76bf61c7a2ce362992)